### PR TITLE
[WNMGDS-630] Update USA banner

### PR DESCRIPTION
**Pull request by nichia**
### Changed
Update USA banner as per USWDS definition.
image001

### How to test
Test the USA banner using various browsers and viewport sizes

### CODE REVIEW

File: packages/design-system-docs/src/styles/components/_UsaBanner.scss [hunk 1]
```diff
@@ -60,10 +60,22 @@
   }
 }
 
+.c-usa-banner__guidance {
+  width: 50%;
```
Comment (bernardwang) [inline packages/design-system-docs/src/styles/components/_UsaBanner.scss:64]: We want to put this under the small breakpoint so its not 50% on small screens.
` @media (min-width: $width-sm) {`

File: packages/design-system-docs/src/styles/components/_UsaBanner.scss [hunk 2]
```diff
@@ -60,10 +60,22 @@
...
+}
+
 .c-usa-banner__icon {
   width: 2.5rem;
 }
 
+.c-usa-banner__lock-image {
+  height: 1.5ex;
+  width: 1.5 * 52 / 64;
```
[the remark this answers is not in the document]
Comment (nichia) [reply packages/design-system-docs/src/styles/components/_UsaBanner.scss:73]: I referenced uswds _banner.scss setting
`.usa-banner__lock-image {
  $lock-h: 64; // unitless height of svg
  $lock-w: 52; // unitless width of svg
  $lock-aspect: $lock-w / $lock-h;
  $icon-height: 1.5ex; // height of the lock icon; use ex for resilience

  height: $icon-height;
  width: $icon-height * $lock-aspect;
  path {
    fill: currentColor;
  }
}`

File: packages/design-system-docs/src/styles/components/_UsaBanner.scss (hunk 1 above)
Comment (nichia) [reply packages/design-system-docs/src/styles/components/_UsaBanner.scss:64]: Ok, i'll check that out.

**Review by bernardwang — APPROVED** [review]
LGTM!

**Pull request merged**